src/Simulation/Native/prerequisites.ps1

```diff
@@ -19,21 +22,25 @@ if (($IsMacOS) -or ((Test-Path Env:AGENT_OS) -and ($Env:AGENT_OS.StartsWith("Dar
     refreshenv
 }
 else {
-    $needClang = !(Get-Command clang-13 -ErrorAction SilentlyContinue)
+    $needClang = !(Get-Command clang-14 -ErrorAction SilentlyContinue)
     if (Get-Command sudo -ErrorAction SilentlyContinue) {
         if ($needClang) {
             wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
-            sudo add-apt-repository "deb https://apt.llvm.org/focal/ llvm-toolchain-focal-13 main"
+            sudo add-apt-repository "deb https://apt.llvm.org/focal/ llvm-toolchain-focal-14 main"
         }
         sudo apt update
-        sudo apt-get install -y clang-13
+        sudo apt-get install -y ninja-build
+        sudo apt-get install -y libomp-14-dev
+        sudo apt-get install -y clang-14
```

Comment:
These can be merged into a single apt-get install call, which is how the QIR Runtime does it.

Reply:
Yes, agreed. But this break-up into multiple lines originates from my observation of one of the intermediate build failures.

There ninja and clang-14 were installed (in this order) by a single `sudo apt-get install -y` command. But the installation of clang-14 was conflicting with the version installed by that moment - clang-13 (see the [parallel PR](https://github.com/microsoft/qdk/pull/274/files)). So the installation of clang-14 was failing and the installation of ninja was not done (or was reverted). So the subsequent steps were complaining for missing ninja instead of wrong clang version.

To make the failures and failure reporting consistent for the described cases I decided to break-up into multiple lines.